src/test/java/edu/kit/compiler/cmd/EchoLineEndingTest.java

```diff
@@ -22,15 +23,24 @@
             "\n" +
             "lkjasdfkjalsdf";
 
+    // ignore this test on windows
     @Test
+    @Disabled
```

Comment:
This is not really related to this PR. Also you can just delete the entire file, as it is not doing anything anymore.